src/app.ts

```diff
@@ -1,18 +1,48 @@
 // Import the express in typescript file
 import express from "express";
 import "dotenv/config";
+import helmet from 'helmet'
+import passport from 'passport'
+import session from 'express-session';
+
+import { passportSetup } from "./authentication/passportSetup";
+
+import { authenticationRoutes } from "./routes/authentication.routes";
+
+
 
 const port = process.env.APP_PORT || 5000;
 
 // Initialize the express engine
 const app: express.Application = express();
 
+// Always use a helmet (Security reasons: https://expressjs.com/en/advanced/best-practice-security.html)
+app.use(helmet());
+app.use(session({
+  secret: 'roll-call-legendary-app', // salt, make session id's harder to crack, value can be anything
```

Comment:
Surely we can put the secret in a .env file? feels dirty to hard code it like this

Reply:
Haha yes, completely forgot about this one, it's named "secret" and I comment it as a thing to make sessions harder to crack. Then leave it exposed 🥇